sudachi/src/input_text/utf8_input_text_builder.rs

```diff
@@ -152,13 +152,11 @@ impl Utf8InputTextBuilder<'_, '_> {
 
             // in rust, char corresponds to unicode scalar value
```

Comment:
I added this comment mentioning to java implementation.
Maybe it's ok to remove it.

Reply:
OK